model/src/main/java/org/eclipse/rdf4j/model/impl/LinkedHashModel.java

```diff
@@ -167,23 +167,6 @@ public void clear() {
 		statements.clear();
 	}
 
-	@Override
-	public boolean remove(Object o) {
```

Comment:
removed this because it's implemented in AbstractModel.java.

Reply:
I notice the implementation is different though. I haven't looked in detail but is the way AbstractModel performs a remove as efficient as the more specific LinkedHashModel implementation?

Reply:
My understanding is that is just converts the Statements into its parts (subject, predicate...) and calls the other remove(...) method with those parts. 

I will double check though.

Reply:
Double checked. Essentially identical.